tests/tests/test_mtls.py

```diff
@@ -274,9 +273,7 @@ def test_mtls_enterprise(self, setup_ent_mtls, algorithm):
         deploy.check_expected_status("finished", deployment_id)
 
         # verify the update was actually installed on the device
-        out = setup_ent_mtls.device.run(
-            "mender show-artifact"
-        ).strip()
+        out = setup_ent_mtls.device.run("mender show-artifact").strip()
```

Comment:
You could use [`clang-format-range`](https://gitlab.com/kacf/gems/-/blob/master/clang-format-range?ref_type=heads) to retroactively format all your commits correctly, instead of fixing it in a dedicated commit. 🙂

Reply:
Nice!